requirements.txt

```
@@ -0,0 +1,3 @@
+pandas>=1.2
+torchvision>=0.9.1
+matplotlib>=3.4.2
```

```suggestion
pandas~=1.2
torchvision~=0.9.1
matplotlib~=3.4
```

As this project is going to be [more of an applicaiton than a library](https://caremad.io/posts/2013/07/setup-vs-requirement/) it is probably worth locking in version ranges beyond just setting lower bounds. I'd suggest doing this with the compatible release syntax of `~=` to specify pinning to different ranges (c.f. https://github.com/mwouts/jupytext/pull/596 for an example).

Here I'm suggesting pinning to the major version (allowing changes in minor and patch releases) for `pandas` and `matplotlib` as the have very matures APIs and do a good job of actually implementing [SemVer](https://semver.org/), and I'm suggesting pinning to the minor release (allowing changes in patch releases) for `torchvision` as it isn't a v1.0 release yet and so under SemVer that's fair game for lots of API changes, but I doubt a patch release will break things given that `torchvision` is a few years old.

We can add some simple CI later to see how implimenting these restrictions is a good way to check for breakage dur to version float.

---

I wasn't aware of the ~= compatible release version specifier so I used the lower bound specifiers. I'll commit these changes and link them to this issue.

---

Yeah, I only learned about them in the last 2 years. There are plenty of people out there that don't know about them or use them yet.